bin/iuf-rpm-upload

```diff
@@ -120,7 +116,7 @@ do
     repo_path=$(echo "$json_rpm_context" | jq -r '.rpms['$i'].path')
     repo_name=$(echo "$json_rpm_context" | jq -r '.rpms['$i'].repository_name')
     repo_type=$(echo "$json_rpm_context" | jq -r '.rpms['$i'].repository_type')
-    echo "Processing RPM content at $repo_path (repo_name=$repo_name repo_type=$repo_type)"
+    echo "INFO Processing RPM content at $repo_path (repo_name=$repo_name repo_type=$repo_type)"
```

Comment:
optimally we'd make this debug, then have an info where we report success of the operation

Reply:
Done